internal/appsec/remoteconfig.go

```diff
@@ -337,13 +336,6 @@ func (a *appsec) enableRemoteActivation() error {
 	if a.rc == nil {
 		return fmt.Errorf("no valid remote configuration client")
```

Comment:
```suggestion
		return fmt.Errorf("appsec: no valid remote configuration client")
```

Sorry I know this isn't part of your changes